Sources/VLCMediaViewEditCell.swift

```diff
@@ -0,0 +1,131 @@
+/*****************************************************************************
+ * VLCMediaViewEditCell.swift
+ *
+ * Copyright © 2018 VLC authors and VideoLAN
+ * Copyright © 2018 Videolabs
+ *
+ * Authors: Soomin Lee <bubu@mikan.io>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+struct VLCCheckView {
+    var isEnabled: Bool {
+        didSet {
+            let backgroundColor: UIColor = isEnabled ? .orange : .clear
+            let borderColor: UIColor = isEnabled ? .clear : .lightGray
+            view.backgroundColor = backgroundColor
+            view.layer.borderColor = borderColor.cgColor
+        }
+    }
+    var view: UIView
+}
+
+class VLCMediaViewEditCell: UICollectionViewCell {
+
+    static let identifier = String(describing: VLCMediaViewEditCell.self)
+
+    static let height: CGFloat = 56
```

Comment:
This should ideally not be necessary and the thumbnail size + contraints to the border should resize the cell as needed